post_office/validators.py

```diff
@@ -1,28 +1,31 @@
+import re
+
 from django.core.exceptions import ValidationError
-from django.core.validators import validate_email
+from django.core.validators import validate_email as validate_plain_email, EmailValidator
 from django.template import Template, TemplateSyntaxError
 from django.utils.encoding import force_text
 
 from .compat import text_type
 
-
-def validate_email_with_name(value):
-    """
-    Validate email address.
+class FullEmailValidator(EmailValidator):
```

Comment:
Let's just restore the `validate_email_with_name` function as it was previously and get rid of the `FullEmailValidator` class. It's cleaner and simpler that way. Thanks!